utils/common.py

```diff
@@ -20,6 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+def get_enum_label(datum, key, Enum):
+    val = datum[key]
+    obj = Enum.get(val)
+    return getattr(obj, "label", val)
```

Comment:
Related to https://github.com/idmc-labs/helix-server/pull/520#discussion_r1346801569
Let's define the function like this, then.
```suggestion
import typing
from django_enumfield import enum


def get_enum_label(enum: enum.Enum, fallback_value: str) -> typing.Union[str, None]:
    """
    Return enum label, use fallback_value if enum is None
    """
    if enum is None:
        return fallback_value
    return enum.label
```